packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts

```diff
@@ -395,6 +399,7 @@ export const getStyles = (props: IChoiceGroupOptionStyleProps): IChoiceGroupOpti
         margin: '4px 8px',
         height: labelWrapperLineHeight * 2,
         lineHeight: labelWrapperLineHeight,
+        maxWidth: imageWidth! * 2, // using non-null assertion because we have a default in `ChoiceGroupOptionBase` class.
```

Comment:
Not commenting on your changes. But I question the original reasoning to set max-width twice of the image width, more so in case of an Icon. Design files shows a 8px padding on both left and right of the label.

Reply:
I tackled only the issue that was presented and think that a style cleanup should be made separate with a better investigation of why this math was done in the first place.